cmd/snap-update-ns/change.go

```diff
@@ -143,7 +151,11 @@ func (c *Change) ensureTarget(sec *Secure) ([]*Change, error) {
 			}
 		}
 	} else if os.IsNotExist(err) {
-		changes, err = c.createPath(path, true, sec)
+		if c.Entry.XSnapdIgnoreMissing() {
```

Comment:
Could you move this inside createPath? You have access to c.Entry there after all. WDYT?